SUPPORTED_LANGUAGES.md

```diff
@@ -190,6 +190,10 @@ The table below shows the full list of languages (and corresponding classes/alia
 | Q                       | k, kdb                 |         |
 | QML                     | qml                    |         |
 | R                       | r                      |         |
+| Raku                    | raku, Perl6, p6, pm6, rakumod | [highlightjs-raku](https://github.com/highlightjs/highlightjs-raku) |
+| RakuDoc                 | pod6,POD6, rakudoc     | [highlightjs-raku](https://github.com/highlightjs/highlightjs-raku) |
+| RakuQuoting             | rakuquoting            | [highlightjs-raku](https://github.com/highlightjs/highlightjs-raku) |
+| RakuRegexe              | rakuregexe, Perl 6, p6, pm6, rakumod | [highlightjs-raku](https://github.com/highlightjs/highlightjs-raku) |
```

Comment:
Aliases should all be lowercase.

Reply:
Is that an issue if they are Capitalized in the project? I copied how it's formatted upstream.

Reply:
> Is that an issue if they are Capitalized in the project? 

We lower-case all aliases internally when registered.  So `Perl6` and `perl6` are technically the same, but we prefer all lower-case, so that's how we will list them in our own docs.